### DISCUSSION TITLE
why i use github in mobile?

### DISCUSSION THREAD

**Discussion by abidhanik** [community/community#65511] [Mobile]
### Select Topic Area

Question

### Body

why i use github in mobile.

**Comment by azmarifdev** [comment]
easy to use.

**Comment by Akash1134** [comment]
GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this post. Please note any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
